crates/sage-api/src/records/offer_summary.rs

```diff
@@ -9,6 +9,9 @@ pub struct OfferSummary {
     pub fee: Amount,
     pub maker: OfferAssets,
     pub taker: OfferAssets,
+    pub expiration_height: Option<u32>,
+    pub expiration_timestamp: Option<u64>,
+    pub expiration_date: Option<String>,
```

Comment:
I think this should be moved to the frontend, so that the backend doesn't have to care about how things are displayed.

Reply:
you mean just `expiration_date`?